stories/storyComponents/LabeledInput.tsx

```diff
@@ -1,6 +1,30 @@
 import * as React from 'react';
-import black from '../../src/colors/black';
-import { spacing } from '../storyStyles';
+import { makeStyles } from '../../src/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(4),
+  },
+  input: {
+    appearance: 'none',
+    borderRadius: '0.25rem',
+    border: `1px solid ${theme.palette.black[100]}`,
+    paddingTop: '0.5rem',
+    paddingBottom: '0.5rem',
+    paddingLeft: '0.75rem',
+    paddingRight: '0.75rem',
+    boxShadow: `
+      0 1px 3px 0 ${theme.hexToRgba(theme.palette.common.black, 0.1)}, 
+      0 1px 2px 0 ${theme.hexToRgba(theme.palette.common.black, 0.06)}
+    `,
+  },
+  label: {
+    display: 'block',
+    marginBottom: theme.spacing(2),
+    fontWeight: 'bold',
```

Comment:
Should we use the theme font weight?
```suggestion
     fontWeight: theme.typography.fontWeightBold,
```

Reply:
Sounds good to me!